### PR TITLE
User profile

### PR DESCRIPTION
**Pull request by m13hall**
now with no debuggers

### CODE REVIEW

File: app/views/profile/show_user.html.erb [hunk 1]
```diff
@@ -46,9 +46,11 @@
       <br/>
       <ul class="list-group">
         <h4>Skills:</h4>
-        <% if !@user.skills.empty? %>
-          <% @user.skills.each do |skill| %>
-            <li class="list-group-item"><%= skill.name %></li>
+        <% if !@user.user_skills.empty? %>
+          <%  @user.user_skills.each do |user_skill| %>
+          <% unless user_skill.experience_level=='None' %>
+            <li class="list-group-item"><%= Skill.all.find(user_skill.id).name %></li>
```
Comment (m13hall) [inline app/views/profile/show_user.html.erb]: This is currently a lot of db hits, even though they'll likely be cached. If anyone knows how to set a global variable for this section that would be better

File: app/views/users/registrations/edit.html.erb [hunk 2]
```diff
@@ -178,18 +178,19 @@
 
 
 <% @skills.each do |skill|
-  if @user_skills.key?(skill.id)
+
+ if @user_skills.has_key?(skill.id)    
     user_skill = @user_skills[skill.id]
   else
-    user_skill = resource.user_skills.new(skill_id: skill.id)
+    user_skill = resource.user_skills.create(skill_id: skill.id, experience_level: "None")
```
Comment (m13hall) [inline app/views/users/registrations/edit.html.erb]: this is the change that makes all users "have" all the skills all the time and just vary the experience level.

File: app/models/skill.rb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 class Skill < ActiveRecord::Base
-  attr_accessible :name, :skill_level
+  attr_accessible :name
```
Comment (schmitzc) [inline app/models/skill.rb]: You should create a migration to drop `skill_level` from `skills` table, too.

**Comment by schmitzc** [comment]
👍 Nice work!

**Pull request merged**